hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/MapInfoCollector.java

```diff
@@ -39,7 +39,13 @@
         Map<String, String> mapInfo = new HashMap<>();
 
         mapInfo.put("mpct", String.valueOf(maps.size()));
+        mapInfo.put("mpbrct", String.valueOf(countMapWithBackupReadEnabled(hazelcastNode)));
 
         return mapInfo;
     }
+
+    private long countMapWithBackupReadEnabled(Node node) {
+        return maps.stream().filter(distributedObject -> node.getConfig().
+                getMapConfig(distributedObject.getName()).isReadBackupData()).count();
```

Comment:
I'm worried about `config.getMapConfig()` returning `null` in some edge cases (eg. if the map exists when we collect the maps from the node, but it is already destroyed when this line gets executed?). So please make this NPE-proof.